### ISSUE TITLE
Should we close this project in favour of GitTools/GitReleaseManager?

### ISSUE THREAD

**Issue by distantcam** [Particular/GitHubReleaseNotes#33]
https://github.com/GitTools/GitReleaseManager

Pros:
- We no longer need to maintain our own version of this tool.
- Configurable so different projects can have different settings.

Cons:
- Not controlled by us, so if we want something custom it may be difficult.

Thoughts?

**Comment by gep13** [comment]
>@distantcam said...
Not controlled by us, so if we want something custom it may be difficult.

This likely wouldn't be an issue.  Assuming it didn't break anything for anyone else, I would likely accept any contributions that you wanted to make.

**Comment by distantcam** [comment]
@gep13 Yep, I just wanted to list it as a potential issue. If worst came to worst we could always fork your fork of our original project. :P

**Comment by gep13** [comment]
>@distantcam said...
fork your fork of our original project. :P

Ha ha! 😄
